src/ExposureHistory/History/index.tsx

```diff
@@ -39,7 +40,10 @@ const History: FunctionComponent<HistoryProps> = ({
     navigation.navigate(Screens.MoreInfo)
   }
 
-  const showExposureHistory = exposures.length > 0
+  const recentExposures = exposures.filter((e) =>
```

Comment:
Nice job! I would think that this gets trimmed from the native side but apparently not.